tests/e2e/cucumber/steps/api.ts

```diff
@@ -82,17 +82,18 @@ Given(
 )
 
 Given(
-  '{string} shares the following resource using API',
+  '{string} shares the following resource using sharingNG API',
   async function (this: World, stepUser: string, stepTable: DataTable): Promise<void> {
     const user = this.usersEnvironment.getUser({ key: stepUser })
     for (const info of stepTable.hashes()) {
-      await api.share.createShare({
+      await api.share.createShareNGForResourceSharing({
```

Comment:
let's use existing name
```suggestion
      await api.share.createShare({
```